speech/gen_dataVals_from_wave_viewer.m

```diff
@@ -107,6 +107,21 @@
             numUserEvents = 0;
         end
 
+        % reorder events first-to-last
+        if numUserEvents >= 2
+            event_times = trialparams.event_params.user_event_times;
+            event_names = trialparams.event_params.user_event_names;
+
+            [~, sortOrder] = sort(event_times(:));
+            if ~isequal(sortOrder', 1:numUserEvents) % events not ordered first-to-last
```

Comment:
Is there a reason to use the `(:)` here? (I think event_times starts out as a row vector but using (:) swaps it to a column vector, which is why you need to transpose it in the next line)

Reply:
Fixed in commit edb007657d445a65824b31428433854f1bbcc1d2